src/log-entry-dashboard/log-entry-form/log-entry-form.component.tsx

```diff
@@ -20,124 +20,305 @@ import {
 import { withAuthenticationContext } from '../../_shared/contexts/authentication/with-authentication-context.component';
 import { LogEntry } from '../../_shared/interfaces/log-entry.interface';
 import { createLogEntry } from '../../_shared/services/log-entry.service';
-import { inputClass } from './log-entry-form.styles';
-import { panelClass } from '../log-entry-dashboard.styles';
+import { timeForm, buttonContainer } from './log-entry-form.styles';
+import { RadioChangeEvent } from 'antd/lib/radio';
+import { any } from 'prop-types';
 
 const FormItem = Form.Item;
+const RadioButton = Radio.Button;
+const RadioGroup = Radio.Group;
+const { TextArea } = Input;
+const SelectOption = Select.Option;
 
 interface LogEntryFormProps extends FormComponentProps {
   authContext: IAuthenticationContext;
 }
 
 class LogEntryFormComponent extends React.Component<LogEntryFormProps> {
+  state = {
+    categoryId: 0,
+    pointsCompeleted: 'notCompleted',
+  };
+
+  handleHourChange = (e?: number) => {
```

Comment:
is this method necessary considering `getFieldDecorator` will control form fields.

Reply:
also, even if it is necessary, `e` is not being used and the value is always being set to `0`

Reply:
Thanks. This function is not necessary and will be deleted.